src/components/header/modals/ModalAccountUnification.vue

```diff
@@ -168,6 +189,10 @@ export default defineComponent({
         if (!success) {
           return;
         }
+      } else if (step === 101) {
+        // Make a call to update unified account identity
+        await updateAccount(unifiedAccount.value.nativeAddress, accountName.value);
```

Comment:
```suggestion
        await Promise.all([ 
          updateAccount(unifiedAccount.value.nativeAddress, accountName.value), 
          await checkIfUnified(unifiedAccount.value.nativeAddress)
        ])
```

Reply:
I can't do that because it can happen that `checkIfUnified` call (which should pick up a new account name) is executed before the account is actually updated with `updateAccount`

Reply:
I see, ok!